### DISCUSSION TITLE
모바일 환경에서의 버튼 입력 안 되는 버그

### DISCUSSION THREAD

**Discussion by FOJF** [naverz/zepeto-studio-kor#369] [5. Q&A]
유니티 환경에서는 버튼의 입력이 아주 잘 되는데 제페토 어플로 테스트 할 때는 버튼을 2~3초 정도 눌렀다가 때야 눌립니다.

원인을 알려주실 수 있을까요?

iOS 15.3, ZEPETO APP 3.9.8, ZEPETO.World 1.1.3입니다.

감사합니다.

**Comment by moondory77-naverz** [comment]
안녕하세요.
문제상황을 조금 더 구체화 해주실 수 있다면, 적확한 도움을 드릴 수 있을 듯 합니다.

일반적인 Button UI 컴포넌트의 EventListener에서 일괄적으로 발생하고 있는 지, 특정 상황에서만 발생하는지,
재현 가능한 스크립트를 첨부해 주시면 더욱 좋을 듯 합니다.

감사합니다.

**Comment by peterkangdev** [comment]
도움이 되실지 모르겠지만, 
저는 UI 이미지에 EventTrigger 를 넣어서 PointerDown 이벤트를 넣어서 사용합니다.
버튼은 님이 말씀하신대로 잘 작동하지 않습니다.

**Reply by moondory77-naverz** [reply]
안녕하세요.
도움이 되실까 하여, 제페토 스크립트의 각종 예제가 포함된 repo를 안내드립니다.
[https://github.com/naverz/zepeto-script-sample](https://github.com/naverz/zepeto-script-sample)

해당 프로젝트의 8.UIEvent에서, 
Button 컴포넌트 사용방법을 확인하실 수 있습니다. 참고부탁드립니다.
